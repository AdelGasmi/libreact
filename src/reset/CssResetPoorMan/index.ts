import createCssResetComponent from '../createCssResetComponent';

export const css = {
  'html,body': {
    pad: 0,
    mar: 0,
  },
  html: {
    fz: '1em',
  },
  body: {
    fz: '100%',
  },
  'a img,a:link img,a:visited img': {
    bd: 0,
  },
};

const CssResetPoorMan = createCssResetComponent(css);

export default CssResetPoorMan;
