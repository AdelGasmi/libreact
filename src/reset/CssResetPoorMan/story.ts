import {createElement as h} from 'react';
import {storiesOf} from '@storybook/react';
import {action} from '@storybook/addon-actions';
import {linkTo} from '@storybook/addon-links';
import CssResetPoorMan from '.';

storiesOf('CSS resets', module)
  .add('CssResetPoorMan', () =>
    h(CssResetPoorMan)
  );
