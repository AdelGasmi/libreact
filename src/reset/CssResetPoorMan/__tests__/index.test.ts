import {createElement as h} from 'react';
import {renderToStaticMarkup} from 'react-dom/server';
import CssResetPoorMan from '..';

describe('<CssResetPoorMan>', () => {
  it('renders as expected', () => {
    const html = renderToStaticMarkup(h(CssResetPoorMan));

    expect(html).toMatchSnapshot();
  });
});
