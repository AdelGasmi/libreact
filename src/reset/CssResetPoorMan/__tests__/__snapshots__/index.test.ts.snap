// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CssResetPoorMan> renders as expected 1`] = `"<style>html,body{padding:0;margin:0;}html{font-size:1em;}body{font-size:100%;}a img,a:link img,a:visited img{border:0;}</style>"`;
